From 1dcf7acb4288a27f125458bfa86cf614437657d5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tiziano=20M=C3=BCller?= <tiziano.mueller@stepping-stone.ch>
Date: Tue, 20 May 2014 18:49:28 +0200
Subject: [PATCH 2/2] Do not depend on libsoup directly

The libsoup-dependency is not directly used but comes in as a dependency
of phodav and phodav has libsoup correctly recorded in its pkg-config
file.
---
 configure.ac | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 70ab75c..f47ee20 100644
--- a/configure.ac
+++ b/configure.ac
@@ -267,9 +267,6 @@ PKG_CHECK_MODULES(GTHREAD, gthread-2.0 > 2.0.0)
 AC_SUBST(GTHREAD_CFLAGS)
 AC_SUBST(GTHREAD_LIBS)
 
-PKG_CHECK_MODULES(SOUP, libsoup-2.4)
-AC_SUBST(SOUP_CFLAGS)
-AC_SUBST(SOUP_LIBS)
 AC_ARG_ENABLE([webdav],
   AS_HELP_STRING([--enable-webdav=@<:@auto/yes/no@:>@],
                  [Enable webdav support @<:@default=auto@:>@]),
-- 
1.9.2

